sbi/utils/dkl.py

```diff
@@ -0,0 +1,33 @@
+import torch
+
+
+def dkl_monte_carlo_estimate(p, q, num_samples: int = 1000) -> torch.Tensor:
+    """
+    Computes the Monte-Carlo estimate of the Kullback-Leibler divergence of two
```

Comment:
Returns... (there is a good reason for this: caller code cares about what it receives back. We try to avoid this, but it is possible also to write functions that modify global variables, typically it is the case for methods modifying class attributes).

Reply:
I agree, it is clear that it computes something. However, it is not clear whether it returns, so "Returns ..." is more informative. Is this fixed?

Reply:
Will also be fixed in next commit (in an hour or so).